OpenProblemLibrary/NAU/setLinearAlgebra/minpoly2.pg

```diff
@@ -26,6 +26,7 @@ TEXT(beginproblem());
 
 Context('Numeric');
 Context()->variables->add(t => ['Real']);
+Context()->variables->remove("x");
```

Comment:
There is a one-step means of doing the two lines above:

```
Context()->variables->are(t => ['Real']);
```

(changing `add` to `are`).